### PR TITLE
Remove test redefining `EMPTY_TRASH_DAYS`.

### LINKED ISSUE #356: Unit test suite throws an error attempting to redefine `EMPTY_TRASH_DAYS`
## Bug Description

If the constant `EMPTY_TRASH_DAYS` is not defined in the config or test config file, WordPress assigns a default value of `30` in the `default-contants.php` file.

https://github.com/WordPress/wordpress-develop/blob/95d26d682fb0fcb77fed308436e87914f7ce2e82/src/wp-includes/default-constants.php#L371-L373

The performance lab test suite attempts to define this value to zero in `WebP_Uploads_Load_Tests::it_should_remove_the_webp_version_of_the_image_if_the_image_is_force_deleted_and_empty_trash_days_is_set_to_zero` causing an error to be thrown:

## Steps to reproduce

1. Remove `EMPTY_TRASH_DAYS` from all test and general config files
2. Run `phpunit --filter force_deleted_and_empty_trash_days_is_set_to_zero` on the command line
3. phpunit should notify you of an error due when attempting to redefine the constant

## Textdumps

```
$ phpunit --filter force_deleted_and_empty_trash_days_is_set_to_zero
Installing...
Running as single site... To run multisite, use -c tests/phpunit/multisite.xml
Not running ajax tests. To execute these, use --group ajax.
Not running ms-files tests. To execute these, use --group ms-files.
Not running external-http tests. To execute these, use --group external-http.
PHPUnit 7.5.20 by Sebastian Bergmann and contributors.

E                                                                   1 / 1 (100%)

Time: 7.8 seconds, Memory: 52.01 MB

There was 1 error:

1) WebP_Uploads_Load_Tests::it_should_remove_the_webp_version_of_the_image_if_the_image_is_force_deleted_and_empty_trash_days_is_set_to_zero
Constant EMPTY_TRASH_DAYS already defined

/vagrant/content/plugins/performance-lab/tests/modules/images/webp-uploads/load-tests.php:279
```

## Additional Context

 - PHP Version: 7.4
 - OS: Ubuntu
 - Browser: N/A
 - Plugin Version: `trunk` at 6c3e79573c39a3a87300c97405ac83f36d0aeee2
 - Device: N/A

I'm running using a https://github.com/Chassis/Chassis box on VirtualBox with the following `config.local.yaml` file:

```yaml
# Domain names to use for Vagrant
hosts:
    - wp-perf.local

# Site Configuration
# (When overriding, include all values)
site:
    name: WP Perf Plugin

# Should we use multisite?
# Values: No, Yes, subdomains
multisite: No

# Plugins
plugins:
   - query-monitor
   - user-switching

# Extensions
extensions:
 - mailhog
 - imagick
 - phpunit
 - xdebug
 - memcache
 - tester

# PHP version
# Values: 5.6, 7.0, 7.1, 7.2, 7.3, 7.4, 8.0 (or 5.6.30)
php: 7.4

phpunit:
  version: 7.5
```

### PR DESCRIPTION
**Pull request by peterwilsoncc**
## Summary

Removes the test `it_should_remove_the_webp_version_of_the_image_if_the_image_is_force_deleted_and_empty_trash_days_is_set_to_zero` to avoid an error being thrown due to redefining a constant.

The test can be deleted in it's entirety as without the `define()` it is identical to `it_should_remove_the_attached_webp_version_if_the_attachment_is_force_deleted_but_empty_trash_day_is_not_defined()` -- renamed in this pull request.

Fixes #356.

## Checklist

- [x] PR has either `[Focus]` or `Infrastructure` label.
- [x] PR has a `[Type]` label.
- [x] PR has a milestone or the `no milestone` label.

_Note: I copied labels over from the issue, please let me know if that is incorrect -- PW_

### CODE REVIEW

**Review by mehulkaklotar — APPROVED** [review]
Looks good! 🚀

**Review by mitogh — APPROVED** [review]

**Review by felixarntz — APPROVED** [review]
Great!
